examples/passport-login/src/authentication-interceptors/facebook-interceptor.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  inject,
+  Provider,
+  Interceptor,
+  InvocationContext,
+  Next,
+} from '@loopback/core';
+import {
+  RestBindings,
+  RequestContext,
+  toInterceptor,
+  ExpressRequestHandler,
+} from '@loopback/rest';
+
+export class FacebookOauthInterceptor implements Provider<Interceptor> {
```

Comment:
Let's mark it with `@bind({scope: BindingScope.SINGLETON})`.

Reply:
agreed